cobra/manipulation/modify.py

```diff
@@ -93,7 +94,8 @@ def rename_genes(cobra_model, rename_dict):
             pass
         else:  # not old gene_present and not new_gene_present
             # the new gene's _model will be set by repair
-            cobra_model.genes.append(Gene(new_name))
+            # cobra_model.genes.append(Gene(new_name)) # MAC Sept 2017 edit: Removed, otherwise, adds genes that are unassigned to reactions
```

Comment:
You can just delete the old line. Git keeps track of the history anyways :)

Reply:
thanks! I'll make the test!